SU-SWS/stanford-jumpstart-deployer
# Security updates: Drupal core, and link

# READY FOR REVIEW


# Summary
- Update Drupal core and link for security updates for Anchorage sites
- Remember when this was a thing? It's still a thing.
- Update to https://github.com/SU-SWS/Stanford-Drupal-Profile/commit/5864c597f37524bb03cc6135bd1edf4214660672

# Needed By (Date)
- Let's say March 7th. Two weeks-ish

# Criticality
- How critical is this PR on a 1-10 scale? 2/10
- We've mitigated the security vulnerabilities on that platform; this is just buttoning things up

# Steps to Test

Run this script with `boom4.sh prod` and ensure that all products build

```
#!/bin/bash

# Usage:
# ./boom4.sh dev
# ./boom4.sh prod
# ./boom4.sh all
if [ $1 == "dev" ] || [ $1 == "all" ]; then
  dr7 make --concurrency=4 make/development/jumpstart-academic.make /tmp/jsa-dev
  dr7 make --concurrency=4 make/development/jumpstart-engineering.make /tmp/jse-dev
  dr7 make --concurrency=4 make/development/jumpstart-plus.make /tmp/jsplus-dev
  dr7 make --concurrency=4 make/development/jumpstart-vpsa.make /tmp/jsvpsa-dev
  dr7 make --concurrency=4 make/development/jumpstart.make /tmp/jsv-dev
fi

if [ $1 == "prod" ] || [ $1 == "all" ]; then
  dr7 make --concurrency=4 make/production/jumpstart-academic.make /tmp/jsa-prod
  dr7 make --concurrency=4 make/production/jumpstart-engineering.make /tmp/jse-prod
  dr7 make --concurrency=4 make/production/jumpstart-plus.make /tmp/jsplus-prod
  dr7 make --concurrency=4 make/production/jumpstart.make /tmp/jsv-prod
fi
```

# Affected Projects or Products
- Anchorage sites

# Associated Issues and/or People
- JIRA ticket: DEVOPS-200

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
